MAUI/Scheduler/date-navigation-restriction.md

```diff
@@ -201,6 +201,29 @@ this.Content = scheduler;
 {% endhighlight %}
 {% endtabs %}
 
+### Show Allowed views
+
+The [SfScheduler](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SfScheduler.html) allows users to toggle the visibility of the allowed views by setting value to the `ShowAllowedViews`. By setting this property to false, users can simplify the interface, making it cleaner and more focused, especially in scenarios where view switching is unnecessary. The default value for the property ShowAllowedViews is `true`.
```

Comment:
@saiganesh-sakthivel  - clearly add AloowedViews **Default** and ShowAllowedViews is false